### DISCUSSION TITLE
24-bit Color Animations

### DISCUSSION THREAD

**Discussion by westobin** [phoebe-project/phoebe2#760] [Questions]
GIF animations are limited to 256 colors, which doesn't look so good on a W-UMa EB.

Is there a built-in way to output animation to a 24-bit color animation... like a movie file (AVI, MOV, etc.)? 

Or is there some other trick that you use to produce high-quality animations? (If so, perhaps a tutorial idea...)  

Thanks!

- Wes

**Comment by kecnry** [comment]
Hi Wes - PHOEBE uses autofig and matplotlib for animations and passes `save_kwargs` on to matplotlib's `animation.save`.  For example: `b.plot(animate=True, save='myfile.gif', save_kwargs={'writer': 'imagemagick'})` from the [animations tutorial](http://phoebe-project.org/docs/latest/tutorials/animations) will request using a specific backend for saving the animation to a gif.  If you pass a filename with the correct extension and an appropriate writer and options that are installed and supported on your specific machine, then it should be possible to export to a wide range of output formats, resolutions, framerates, etc.... but you might need to dig into the matplotlib documentation and use trial-and-error to get it working based on which dependencies you have installed on your machine (ffmpeg is a very flexible writer, but might not have it installed by default).  

If you find some good resources and want to put together a tutorial, I agree that would be a great addition (we'll just need to explain in the tutorial that the same arguments might not produce the same results on different machines depending on the setup).
